htpy/__init__.py

```diff
@@ -188,6 +188,8 @@ def _iter_node_context(x: Node, context_dict: dict[Context[t.Any], t.Any]) -> It
         yield from _iter_node_context(x.func(context_value), context_dict)
     elif isinstance(x, str | _HasHtml):
         yield str(_escape(x))
+    elif isinstance(x, int):
+        yield str(x)
```

Comment:
I'm not sure why but this breaks the test. My editor shows the else-branch as unreachable code 🤔

Reply:
The reason is that `bytes` are iterable.